_partials/_devops-cli-reference.md

```diff
@@ -0,0 +1,78 @@
+|         | start http                                         | Start the $MCP_LONG with HTTP transport. Includes flags: `--port` (default: 8080), `--host` (default: localhost)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+
+
+## Global flags
+
+You can use the following global flags with $CLI_LONG:
+
+<GLOBALFLAGS />
+
+
+## Configuration parameters
+
+By default, $CLI_LONG stores your configuration in `~/.config/tiger/config.yaml`. The name of these
+variables matches the flags you use to update them. However, you can override them using the following
+environmental variables:
+
+- **Configuration parameters**
+    - `TIGER_CONFIG_DIR`: path to configuration directory (default: `~/.config/tiger`)
+    - `TIGER_API_URL`: $REST_LONG base endpoint (default: https://console.cloud.timescale.com/public/api/v1)
+    - `TIGER_CONSOLE_URL`: URL to $CONSOLE (default: https://console.cloud.timescale.com)
+    - `TIGER_GATEWAY_URL`: URL to the $CONSOLE gateway (default: https://console.cloud.timescale.com/api)
+    - `TIGER_DOCS_MCP`: enable/disable docs MCP proxy (default: true)
+    - `TIGER_DOCS_MCP_URL`: URL to the MCP server for $COMPANY docs (default: https://mcp.tigerdata.com/docs)
+    - `TIGER_PROJECT_ID`: ID for the project updated when you call $CLI_SHORT commands
+    - `TIGER_SERVICE_ID`: ID for the service updated when you call $CLI_SHORT commands
+    - `TIGER_OUTPUT`: output format (`json`, `yaml`, or table)
+    - `TIGER_ANALYTICS`: enable or disable analytics (default: true)
+    - `TIGER_PASSWORD_STORAGE`: password storage method (keyring, pgpass, or none)
+    - `TIGER_DEBUG`: enable/disable debug logging (default: false)
+
+- **Authentication parameters**
+
+  To authenticate without using the interactive login, set the following with your [client credentials][rest-api-credentials]:
+
+    - `TIGER_PUBLIC_KEY`: public key
+    - `TIGER_SECRET_KEY`: secret key
```

Comment:
To be clear, you still need to run `tiger auth login` when setting these parameters, and you also need to set `TIGER_PROJECT_ID`. For example, you could do:

```bash
TIGER_PUBLIC_KEY=<public_key> TIGER_SECRET_KEY=<secret_key> TIGER_PROJECT_ID=<project_id> tiger auth login
```

This would log you in without performing the interactive OAuth flow.

Alternatively, you could provide the values as flags:

```bash
tiger auth login --public-key=<public_key> --secret-key=<secret-key> --project-id=<project_id>
```